README.md

```diff
@@ -105,6 +134,32 @@ $ cat > /etc/cdi/vendor.json <<EOF
 }
 EOF
 
+### Using Annotations per device to add meta-information
+
+```bash
+$ mkdir /etc/cdi
+$ cat > /etc/cdi/vendor-annotations.json <<EOF
```

Comment:
Probably good to mention that we support both "global" and "per device" annotations, and that these annotations are local to CDI json, meaning not affecting containerEdits parts (which can be separate topic we need to discuss).

Reply:
Are we? The annotations are currently only placed at the device level, I wanted to add annotations also at the spec level but was unsure if needed. 

Are you suggesting to add it here as well: 

```diff

// Spec is the base configuration for CDI
type Spec struct {
	Version string `json:"cdiVersion"`
	Kind    string `json:"kind"`
+       Annotations    map[string]string `json:"annotations,omitempty"`
	Devices        []Device       `json:"devices"`
	ContainerEdits ContainerEdits `json:"containerEdits,omitempty"`
}

```

Reply:
I may have also a  use-case for annotations in the containerEdits, again with Kata, depending on the CDI HW I am exposing I may want to add an containerEdit annotation to trigger a specific behaviour in Kata.

Reply:
> Are we? The annotations are currently only placed at the device level, I wanted to add annotations also at the spec level but was unsure if needed.

in `schema/schema.json` I see it added to two places in this PR. In Go structs only in one place.

Reply:
Ding it, you're right. Let me add it back to the Go struct.

Reply:
The first thought was to add it globally and locally per device and then added it only to the device. I think globally and per devices makes sense. Yeah let's keep the annotation in the containerEdits for a different discussion.